aws/bin/update-repos

```diff
@@ -17,24 +17,6 @@ if ! gpg --list-secret-key opensource+hhvm@fb.com >/dev/null 2>/dev/null; then
   aws kms decrypt --ciphertext-blob "fileb:///opt/hhvm-packaging/aws/gpg-key.kms-ciphertext" --query Plaintext --output text | base64 --decode | gpg --import
 fi
 
-MOUNTED=""
-if [ ! -e /mnt/dl.hhvm.com ]; then
-  # See http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-data-retrieval
-  INSTANCE_ID=$(curl http://169.254.169.254/latest/meta-data/instance-id)
-  # We keep a shared XFS volume as the source of truth to avoid having to
-  # redownload from S3 every day, which can take a *really* long time.
-  VOLUME_ID="vol-096d2c45d13d3a865"
-  aws ec2 attach-volume \
-    --device /dev/sdf \
-    --instance-id "$INSTANCE_ID" \
-    --volume-id "$VOLUME_ID"
-  sleep 10 # wait for EC2 to attach
-
-  mkdir /mnt/dl.hhvm.com
-  mount /dev/xvdf /mnt/dl.hhvm.com
-  MOUNTED="1"
-fi
```

Comment:
this change looks unrelated?

Reply:
Same as the other changes -- this conditional branch of the script was only reached from the old build system. The new one mounts the drive from the [init script](https://github.com/hhvm/packaging/blob/master/aws/hhvm1/worker/init/update-repos.sh) (no need to unmount and re-mount it between each version being published, especially since that is the part that used to randomly fail from time to time).